components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDO.java

```diff
@@ -89,6 +89,7 @@ public class OAuthAppDO extends InboundConfigurationProtocol implements Serializ
     private String subjectType;
     private String requestObjectEncryptionAlgorithm;
     private String requestObjectEncryptionMethod;
+    private boolean isFAPIConformant;
```

Comment:
Don't we need to update the wsdl file related to this?

Reply:
Updated